features/admin.branding.v1/components/custom-text/custom-text-fields.tsx

```diff
@@ -155,6 +163,107 @@ const CustomTextFields: FunctionComponent<CustomTextFieldsProps> = (props: Custo
         );
     };
 
+    const handleMultiAttributeLink = (): void => {
```

Comment:
Shall we separate this logic out to decorators?

Reply:
Will address in a followup PR